ambuda/utils/cheda.py

```diff
@@ -18,56 +18,56 @@ class Token:
 
 
 POS = {
-    "n": "noun",
-    "a": "adjective",
-    "v": "verb",
-    "i": "indeclinable",
-    "va": "participle",
-    "vi": "verbal indeclinable",
+    "n": {"en": "noun", "sa": "subanta"},
+    "a": {"en": "adjective", "sa": "viśeṣaṇa"},
+    "v": {"en": "verb", "sa": "tiṅanta"},
+    "i": {"en": "indeclinable", "sa": "avyaya"},
+    "va": {"en": "participle", "sa": "participle"},
```

Comment:
_kr̥dantam_ is probably acceptable here. Sadly our parse data isn't rich enough to help us say more, but we'll get there in time.